src/Data/Product.agda

```diff
@@ -113,6 +114,12 @@ map₂ : ∀ {A : Set a} {B : A → Set b} {C : A → Set c} →
        (∀ {x} → B x → C x) → Σ A B → Σ A C
 map₂ f = map id f
 
+-- A version of map where the output can depend on the input
```

Comment:
all the changes in `Data.Product` (which are great) are not in the `CHANGELOG`.

Reply:
I'll get onto this now...